docs/knowledgebase/getting-started/index.md

```diff
@@ -74,119 +74,69 @@ rustup default stable
 
 Substrate uses [WebAssembly](https://webassembly.org/) (Wasm) to produce portable blockchain
 runtimes. You will need to configure your Rust compiler to use
-[`nightly` builds](https://doc.rust-lang.org/book/appendix-07-nightly-rust.html) to allow
-compiled substrate compatible runtimes to Wasm.
+[`nightly` builds](https://doc.rust-lang.org/book/appendix-07-nightly-rust.html) to allow you to
+compile Rust code to the Wasm target.
```

Comment:
You can compile Rust code without nightly to Wasm. It is the substrate runtime that requires nightly because of design decisions we made: https://github.com/paritytech/substrate/issues/1252